packages/backend/src/functions/httpTrigger.ts

```diff
@@ -130,10 +131,14 @@ async function getProvenance(request: HttpRequest, context: InvocationContext):
         const { data, timestamp } = await decryptBlob(blobClient, deviceKey);
         const json = new TextDecoder().decode(data);
         const provRecord = JSON.parse(json) as ProvenanceRecord;
+        const deviceID = await calculateDeviceID(deviceKey);
+        provRecord.record.deviceID = deviceID;
         records.push({ ...provRecord, timestamp });
     }
     records.sort((a, b) => b.timestamp - a.timestamp)
-    return { jsonBody: records };
+  return {
+    headers: { "Access-Control-Allow-Origin" : "*"},

```

Comment:
CORS is configured at the [service level](https://learn.microsoft.com/en-us/azure/azure-functions/functions-how-to-use-azure-function-app-settings?tabs=portal#cors), so we shouldn't need this.

Reply:
Removing line 140 DEFINITELY causes a CORS failure in this branch:


![Screen Shot 2024-03-19 at 11 49 00 AM](https://github.com/gosqasorg/asset-provenance-tracking/assets/5296671/8b4c222c-7f05-4378-9b68-840a85fecffb)

Reply:
CORS can be configured when hosting locally via the [local.settings.json file](https://stackoverflow.com/a/55875060). I think we should configure this once in the settings file and remove the explicit code in the httpTriggers.ts file

Reply:
I have done as you suggested and tested it.